<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>init_widget</class>
 <widget class="QWidget" name="init_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>165</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>228</width>
    <height>165</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>228</width>
    <height>165</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>                   Prepare for game !</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="client_name"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="server_button">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Client</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="client_button">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Server IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="server_ip"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="start_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Start!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
